Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -112,22 +112,36 @@ configuration:
   required: false
   type: 15
   section: Collect
-- additionalinfo: When selected, closing the Splunk notable event is mirrored in Cortex XSOAR.
+- additionalinfo: If selected, closing a Notable Event on Splunk with a status of "Closed" will close the corresponding fetched incident on XSOAR.
   defaultvalue: 'false'
-  display: Close Mirrored XSOAR Incident
+  display: Mirror closed XSOAR Incidents (Incoming Mirroring)
   name: close_incident
   required: false
   type: 8
   section: Collect
   advanced: true
-- additionalinfo: When selected, closing the Cortex XSOAR incident is mirrored in Splunk.
+- additionalinfo: If selected, Notables with a status label that is marked as "End Status" on Splunk will also be mirrored on closure.
   defaultvalue: 'false'
-  display: Close Mirrored Splunk Notable Event
+  display: Add Splunk "End Status" statuses to mirroring (Incoming Mirroring)
+  name: close_end_status_labels
+  required: false
+  type: 8
+  section: Collect
+  advanced: true
+- additionalinfo: "A comma-separated list of status labels to add to mirroring (Example: Resolved,False-Positive)."
+  display: Additional Splunk status labels to mirror (Incoming Mirroring)
+  name: close_extra_labels
+  required: false
+  type: 0
+  section: Collect
+- display: Mirror Closed Splunk Notable Events (Outgoing Mirroring)
   name: close_notable
   required: false
   type: 8
   section: Collect
   advanced: true
+  additionalinfo: If selected, closing a fetched incident of a Splunk Notable on XSOAR will close the corresponding Notable Event on Splunk.
```

Comment:
If selected, closing a fetched incident of a Splunk Notable in Cortex XSOAR will close the corresponding Notable Event in Splunk.